cmd/server/assets/codestatus/index.html

```diff
@@ -50,7 +50,7 @@ <h1>Verification code status</h1>
     <div class="card mb-3 shadow-sm">
       <div class="card-header">Recently issued codes</div>
       <div class="card-body">
-        <div class="list-group">
+        <div class="list-group" id="recent-codes">
```

Comment:
@whaught there's actually a `data-datetime` attribute that will do this automagically and accounts for some edges (see header.html)

Reply:
Done. Although I did kind of think the `PST` was a little reassuring